### ISSUE TITLE
FN - Terraform with S3 Bucket ACL Allows Read and write to Any  User configuration is not detected

### ISSUE THREAD

**Issue by securylight** [Checkmarx/kics#3772]
### Expected Behavior
S3 Bucket ACL  that Allows Read and write to Any User Terraform configuration should be detected.
aka when  acl    = "public-read /  "public-read-write" / "authenticated-read"

### Actual Behavior
S3 Bucket ACL  that Allows Read and write to Any User Terraform configuration is not detected.
### Steps to Reproduce the Problem

Scan a folder that contains the attached file (pay attention I wrote all the problematic acls separted with coma, there should ve only one.
[s3bucketGet.tf.txt](https://github.com/Checkmarx/kics/files/6733099/s3bucketGet.tf.txt)

Results are attached as well. 
[result.html.txt](https://github.com/Checkmarx/kics/files/6731698/result.html.txt)

Also a screenshot from AWS.
![b_aws](https://user-images.githubusercontent.com/37718207/123762019-0a6fec80-d8cb-11eb-895c-c26f053c859f.png)

### Specifications

-   Version: v1.3.2
-   Platform: Terraform
-   Subsystem:

**Comment by felipe-avelar** [comment]
@securylight I see you are using a module block. Right at this moment, KICS does not supports modules blocks. This will be implemented in the future, but, if it is ok, feel free to help us.

**Comment by felipe-avelar** [comment]
Hello, @securylight, I hope you are doing well. After looking your example and trying to detect the problem, I found out that the sample you provided contain some problems.

on line:
```hcl
acl    = "public-read /  "public-read-write" / "authenticated-read"
```

It is missing a `"`, but, even correcting this typo to:
```hcl
acl    = "public-read" /  "public-read-write" / "authenticated-read"
```
This is not valid, because `/` as a arithmetic operator, as you can see [here](https://www.terraform.io/docs/language/expressions/operators.html#arithmetic-operators) and expects numbers, instead pf strings. If we try to apply this tf file, we will receive the following error:

> Unsuitable value for left operand: a number is required.

Because of this, I'm closing this issue, since it is not a bug.

**Comment by securylight** [comment]
@felipe-avelar please reopen. I mentioned in the Steps to Reproduce the Problem: "pay attention I wrote all the problematic acls separated with coma, there should be only one."

I've tried to run the 3 different terraform files one with public-read, one with public-read-write, one with authenticated-read.

**Comment by rogeriopeixotocx** [comment]
@securylight @felipe-avelar We don't support terraform modules yet. 
This issue can remain open to signal us the need to support this feature.
